metadata.rb

```diff
@@ -4,7 +4,7 @@
 license           'Apache 2.0'
 description       'Manages client.rb configuration and chef-client service'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '4.3.0'
+version           '4.4.0'
```

Comment:
Can you avoid updating version numbers in metadata? I'll do that when ready to release.